PyInstaller/utils/hooks/conda.py

```diff
@@ -39,11 +39,11 @@
 import sys
 from pathlib import Path
 import json
+import fnmatch
 
 from PyInstaller import compat
 from PyInstaller.log import logger
 
-
```

Comment:
non-functional change

Reply:
IMO removing a blank line isn't actually an issue - it doesn't show up in the blame.

Reply:
And anyway, You're reviewing a single commit - it's better to wait until the PR is done, and then nitpick on non-functional changes after all the functional ones are made. (And often commits get squashed together, eliminating some changes totally)